src/intents.cs

```diff
@@ -2364,13 +2883,25 @@ interface INListRideOptionsIntentResponse {
 	[DisableDefaultCtor]
 	interface INMessage : NSCopying, NSSecureCoding {
 
-		[Export ("initWithIdentifier:content:dateSent:sender:recipients:")]
+		[Watch (4,0), Mac (10,13, onlyOn64:true), iOS (11,0)]
+		[Export ("initWithIdentifier:conversationIdentifier:content:dateSent:sender:recipients:groupName:messageType:")]
```

Comment:
that make sense, the new API has an extra argument so it likely calls the _old designated_ init, which makes it non-designated for subclassing.